umm-spec-lib/src/cmr/umm_spec/validation/umm_spec_validation_core.clj

```diff
@@ -66,6 +66,27 @@
     (validation-errors->path-errors
      (v/validate vg/granule-validations granule-with-parent))))
 
+(defn validate-service
+  "Validates the UMM record returning a list of error maps containing a path
+  through the UMM model and a list of errors at that path. Returns an empty
+  sequence if it is valid."
+  ([service]
+   (validate-service service nil))
+  ([service additional-validations]
+   (validation-errors->path-errors
+    (v/validate (cons {} additional-validations)
+                service))))
+
+(defn validate-service-warnings
+  "Validates the UMM record against the list of warnings - issues that we want
+  to convey to the user, but which we do not consider failures."
+  ([service]
+   (validate-service-warnings service nil))
+  ([service additional-validations]
+   (validation-errors->path-errors
+    (v/validate (cons {} additional-validations)
+                service))))
```

Comment:
If supplied the same `additional-validations`, these functions are identical. Is this duplication unnecessary?

Reply:
I don't know, I'm doing what all the other formats are doing.

Reply:
dropping function, tests passed without it.